### PR TITLE
With Python3.5 you can only decrypt bytes.

### PR DESCRIPTION
**Pull request by Natim**

### CODE REVIEW

File: kinto_portier/views.py [hunk 1]
```diff
@@ -141,7 +141,7 @@ def portier_verify(request):
 
     # Store the encrypted user ID with the token
     session_ttl = portier_conf(request, 'session_ttl_seconds')
-    request.registry.cache.set('portier:' + userID, encrypted_email, session_ttl)
+    request.registry.cache.set('portier:' + userID, encrypted_email.decode('utf-8'), session_ttl)
```
Comment (Natim) [inline kinto_portier/views.py]: This is especially important when we are using the PostgreSQL cache backend rather than the memory one.

**Comment by Natim** [comment]
Refs https://github.com/Kinto/kinto/pull/1143/files

File: kinto_portier/views.py (hunk 1 above)
Comment (leplatrem) [inline kinto_portier/views.py]: nit: use intermediary variable `key`

File: kinto_portier/authentication.py [hunk 2]
```diff
@@ -66,7 +66,7 @@ def _verify_token(self, user_token, request):
         if encrypted_email is None:
             return None
 
-        email = decrypt(encrypted_email, user_token)
+        email = decrypt(encrypted_email.encode('utf-8'), user_token)
```
Comment (leplatrem) [inline kinto_portier/authentication.py]: Since `decrypt` and `encrypt` are implement locally in `crypto.py`, wouldn't you want to cast there?

Comment (Natim) [inline kinto_portier/authentication.py]: We probably could.

**Pull request merged**